### ISSUE TITLE
Code insights: link bars of grouping insight type to searches

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#39958]
Blocked by https://github.com/sourcegraph/sourcegraph/issues/39956

Based on [RFC jobs-to-be-done](https://docs.google.com/document/d/1yLV7MGOEXYT3yPvBxHHm8y2-p737KIrju_QPqlhBYz4/edit#heading=h.trqab8y0kufp) we should support URLs for aggregations bar clicks. This means that we should also support links for compute-powered insight on the dashboard page. Currently, API doesn't have all the needed information to achieve this. After API supports this, we should update the FE part accordingly to support links. 

Note: capture group is potentially out of scope for this one. See #29957 for the context.

/cc @joelkw @felixfbecker @vovakulikov @unclejustin

**Comment by vovakulikov** [comment]
@chwarwick based on [this PR](https://github.com/sourcegraph/sourcegraph/pull/40375#discussion_r946697310), we are going to have a link for the aggregation chart on the search result page. But we also should support this functionality for the compute-powered insight if we want to support links for charts on the dashboard page. Do you think we could fit this in the sourcegraph 4.0 or it's out of scope for the first version?

**Comment by vovakulikov** [comment]
cc @Joelkw (mention you just to check that we're ok with moving out of scope for 4.0 clickable bars on the dashboard page)

**Comment by Joelkw** [comment]
Got it, so the bars are clickable on the search page but not when viewing on the dashboard page or single insight view page? 

That's acceptable (but of course not optimal) given the timeline.

**Comment by Joelkw** [comment]
@vovakulikov can you please create the separate issue for that followup work and propose it for iteration 26 (post 4.0)?

**Comment by vovakulikov** [comment]
> Got it, so the bars are clickable on the search page but not when viewing on the dashboard page or single insight view page?

Correct

I don't think we need a separate issue for it. Since link generation will be supported on the backend, this issue will cover follow-up work for compute-powered insight.

**Comment by felixfbecker** [comment]
@sourcegraph/code-insights-frontend I don't think this issue was solved for saved grouping insights in https://github.com/sourcegraph/sourcegraph/pull/40803 (nor was that still in scope before 4.0)
